### PR TITLE
Announce reactive programming course

### PR DESCRIPTION
**Pull request by julienrf**
/cc @rkuhn and @ktoso

### CODE REVIEW

File: news/_posts/2018-12-20-programming-reactive-systems-course.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  [Reactive Manifesto](https://www.reactivemanifesto.org/),
+- **Konrad Malawski**, long-time Akka team and
+  [Reactive Streams](http://www.reactive-streams.org/) expert group member.
+
+In case you have followed the former “Principles of Functional Reactive
+Programming in Scala” course a couple of years ago, this course contains updated
+material as well as three weeks of new material on Akka Typed and Akka Streams.
+
+The first session will start on February 18th, 2019! You can sign up and audit the
+course for free or pursue a certificate.
```
Comment (darjutak) [inline news/_posts/2018-12-20-programming-reactive-systems-course.md]: Maybe put the sign-up link here too?

File: news/_posts/2018-12-20-programming-reactive-systems-course.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+layout: inner-page-no-masthead
+post-type: announcement
+title: "New Course: “Programming Reactive Systems”"
+by: Julien Richard-Foy
+includeTOC: false
+---
+
+We are extremely excited to announce the release of a new MOOC:
```
Comment (olafurpg) [inline news/_posts/2018-12-20-programming-reactive-systems-course.md]: s/extremely//

File: news/_posts/2018-12-20-programming-reactive-systems-course.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+[Programming Reactive Systems](https://www.edx.org/course/programming-reactive-systems)!
+
+This seven weeks long course teaches how to use “reactive programming” techniques
```
Comment (olafurpg) [inline news/_posts/2018-12-20-programming-reactive-systems-course.md]: teaches **you**?

File: news/_posts/2018-12-20-programming-reactive-systems-course.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+for implementing scalable, resilient and responsive systems. Skills you will learn
+include:
```
Comment (olafurpg) [inline news/_posts/2018-12-20-programming-reactive-systems-course.md]: include **how to**?

File: news/_posts/2018-12-20-programming-reactive-systems-course.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- use, transform and sequence asynchronous computations using `Future` values,
+- write concurrent systems based on [Akka](https://akka.io) actors,
+- design systems resilient to failures,
+- implement systems that can scale out according to a varying workload,
```
Comment (olafurpg) [inline news/_posts/2018-12-20-programming-reactive-systems-course.md]: s/a varying workload/varying workloads/

Comment (olafurpg) [reply news/_posts/2018-12-20-programming-reactive-systems-course.md]: s/that can scale/that scale/

File: news/_posts/2018-12-20-programming-reactive-systems-course.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+- **Roland Kuhn**, former Akka team lead and co-author of the
+  [Reactive Manifesto](https://www.reactivemanifesto.org/),
+- **Konrad Malawski**, long-time Akka team and
+  [Reactive Streams](http://www.reactive-streams.org/) expert group member.
+
+In case you have followed the former “Principles of Functional Reactive
+Programming in Scala” course a couple of years ago, this course contains updated
+material as well as three weeks of new material on Akka Typed and Akka Streams.
+
+The first session will start on February 18th, 2019! You can sign up and audit the
```
Comment (olafurpg) [inline news/_posts/2018-12-20-programming-reactive-systems-course.md]: remove exclamation mark?

**Comment by julienrf** [comment]
Thanks for the review, mates!

**Pull request merged**